Formula/llvm.rb

```diff
@@ -225,19 +282,19 @@ def install
   end
 
   def caveats
-    s = <<-EOS.undent
+    caveat_message = <<-EOS.undent
       LLVM executables are installed in #{opt_bin}.
-      Extra tools are installed in #{opt_share}/llvm.
+    Extra tools are installed in #{opt_share}/llvm.
```

Comment:
No, `#{opt_share}/llvm` should be replaced with `#{opt_pkgshare}` (it is defined as `share` + formula name).